ui/package.json

```diff
@@ -18,11 +18,13 @@
     "@types/react-router-bootstrap": "^0.26.6",
     "@types/react-router-dom": "^5.3.3",
     "@types/yup": "^0.32.0",
-    "axios": "^1.6.5",
+    "axios": "0.27.2",
```

Comment:
I was getting an error like described here: https://github.com/openapistack/openapi-client-axios/issues/172
Turns out, this is an incompatibility issue related to create-react-app.

I had it fixed by downgrading axios' version, as suggested here: https://github.com/facebook/create-react-app/discussions/12823#discussioncomment-4295231

In the long run, we might think about migrating this project out of CRA and into Vite-backed codebase, as suggested in the first link.

Reply:
hm that's from 2022, shouldn't we be concerned that we're using such an old dependency? CRA maybe needs to be updated?

Reply:
ah you write about that. Probably this vite thing should happen sooner rather than later?